kolibri/core/assets/src/views/points-icon/index.vue

```diff
@@ -0,0 +1,51 @@
+<template>
+
+  <div :style="dimensions">
+    <file-svg v-if="active" src="./icons/leaf.svg"/>
+    <file-svg v-else src="./icons/grey-leaf.svg"/>
+  </div>
+
+</template>
+
+
+<script>
+
+  function widthHeightValidate(units) {
+    return units.search(/[0-9]+(em|px|%)/) > -1;
+  }
+
+  module.exports = {
+    props: {
+      active: {
+        type: Boolean,
+        required: true,
+      },
+      height: {
+        type: String,
+        required: true,
+        validator: widthHeightValidate,
+      },
+      width: {
+        type: String,
+        required: true,
+        validator: widthHeightValidate,
+      },
```

Comment:
My intuition is that we shouldn't be using props as a way to pass around style parameters if at all possible. A preferable interface would be that the parent component could simply set styles directly on `points-icon` from the outside. E.g.

```html
<points-icon :active="true" style="width: 100px; height: 20em" />
```
or 

```html
<points-icon :active="true" class="thisPointsIcon" />
```